pytak/classes.py

```diff
@@ -166,7 +167,7 @@ async def run(self, number_of_iterations=-1) -> None:
         self._logger.info("Run: %s", self.__class__)
 
         while 1:
-            await asyncio.sleep(0)
+            await asyncio.sleep(self.min_period)
             if self.reader:
```

Comment:
@ampledata does self.reader return true if there are bytes on the socket? If so would it be better to instead replace it with 
```
while self.reader:
    cot = self.read_cot()
```
That way the min period could still be quite large without a noticeable latency.